backend/api/appsettings.Development.json

```
@@ -19,5 +19,13 @@
     }
   },
   "AllowedHosts": "*",
-  "AllowedOrigins": "http://localhost:3001"
+  "AllowedOrigins": "http://localhost:3001",
+  "Mqtt": {
+    "Host": "localhost",
+    "Port": 1883,
+    "Username": "mosquitto",
+    "Topics": [
+      "#"
+    ]

```

```suggestion
    "Topics": [
      "isar/+/mission",
      "isar/+/task",
      "isar/+/step"
    ]

```
I would suggest this instead.